int32 id
uint32 max_markers
float64 confidence

Box bounding_box
Marker[] markers
